packages/client/src/Aigur.ts

```diff
@@ -1,4 +1,5 @@
-import { APIKeys, PipelineConf } from './types';
+import { APIKeys, FlowBuilder as FlowBuilder2, PipelineConf } from '@aigur/types';
```

Comment:
I think, in tearms of readability, a more descriptive name such as `FlowBuilderType` would be better. What do you think?

Reply:
Yeah of course :) It's just a POC PR.
Going to close this one in favor of a real future PR.

Would love to see you contribute to the project!
I'll write a Contribution guide soon to help onboard people that want to get their hands dirty.